internal/notifier/alertmanager.go

```diff
@@ -82,6 +84,7 @@ func NewAlertmanager(hookURL string, proxyURL string, certPool *x509.CertPool) (
 		URL:      hookURL,
 		ProxyURL: proxyURL,
 		CertPool: certPool,
+		Token: token,
```

Comment:
```suggestion
		Token:    token,
```

We use [`go fmt`](https://go.dev/blog/gofmt) to format files, struct fields are always aligned using whitespaces.